tensorflow/python/ops/special_math_ops.py

```diff
@@ -94,7 +94,9 @@ def einsum(axes, *inputs):
   """
   A generalized contraction between tensors of arbitrary dimension.
 
-  Like numpy.einsum.
+  Like numpy.einsum, but does not support:
+  -- ellipses (subscripts like 'ij...,jk...->ik...')
```

Comment:
Can you turn those apostrophes into backticks? Then it'll render nicely as code. Also, please put `numpy.einsum` into backticks. All docstrings get rendered as markdown.